iOS/Views/NewsArticleOverviewCell.swift

```diff
@@ -15,7 +15,6 @@ class NewsArticleOverviewCell: UITableViewCell {
     func configure(for newsArticle: NewsArticle) {
         self.title.text = newsArticle.title
         self.timeSinceCreated.text = NewsArticle.displayDateFormatter.string(for: newsArticle.displayAt)
-        self.content.text = newsArticle.content.convertedHTML?.string.replacingOccurrences(of: "\n", with: " ")
+        self.content.text = newsArticle.content.convertedHTML?.string.removeCharacters(set: .newlines)
```

Comment:
Watch out! You first create an attributedString and derive a string from it afterwards. The functionality exists to get the string directly.